lib/travis/api/app/base.rb

```diff
@@ -1,4 +1,5 @@
 require 'sinatra/base'
+require 'appsignal/integrations/sinatra'
```

Comment:
I would guard this require with an env var check, as I believe this will activate appsignal for all dynos otherwise.

Perhaps we can go with `APPSIGNAL_EANBLED_FOR_DYNOS` for consistency with other similar env vars. e.g. https://github.com/travis-ci/travis-api/blob/master/lib/conditional_skylight.rb#L38